README.md

```diff
@@ -1,2 +1,9 @@
-# publisher-chart
-Libero Publisher vanilla version Helm chart
+Libero Publisher vanilla Helm chart
+====================
```

Comment:
```suggestion
==================================
```

(Think that's the right number. 😅)

Reply:
Why not just `#` forgoing the maintenance of `=` signs?

Reply:
Easier to read.